config/settings.yml.example

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
so this file is not config for core, I suppose we need one for smart_proxy_dynflow with
`:core_url: http://localhost:8008` and standard options like enabled